### PR TITLE
Weatherflow Icon mappings

### PR DESCRIPTION
**Pull request by jeeftor**
## Proposed change

Adding a PR to change how icons are mapped based on (https://github.com/home-assistant/core/issues/113310#issuecomment-2028098397). This documentation addresses the change

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/114497
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #https://github.com/home-assistant/core/issues/113310#issuecomment-2028098397

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/weatherflow_cloud.markdown [hunk 1]
```diff
@@ -18,3 +18,31 @@ ha_integration_type: integration
 The **WeatherFlow Cloud** integration provides access to cloud provided Weather Forecast of a user's Tempest Weather Stations. In order to access the station you will need to configure the integration with an [Api Key](https://weatherflow.github.io/Tempest/api/).
 
 {% include integrations/config_flow.md %}
+
+
+## Weather Icons
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
## Weather icons
```

File: source/_integrations/weatherflow_cloud.markdown [hunk 2]
```diff
@@ -18,3 +18,31 @@ ha_integration_type: integration
...
+
+There is not a straight 1-1 mapping between the Home Assistant supported weather conditions and what tempest provides - as such you may see a difference between what is displayed in the tempest app and what is displayed in Home Assistant.
+
+| Weather Flow Icon | Home Assistant Icon |
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
| Weather Flow icon | Home Assistant icon |
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) also in headings.

File: source/_integrations/weatherflow_cloud.markdown [hunk 3]
```diff
@@ -18,3 +18,31 @@ ha_integration_type: integration
 The **WeatherFlow Cloud** integration provides access to cloud provided Weather Forecast of a user's Tempest Weather Stations. In order to access the station you will need to configure the integration with an [Api Key](https://weatherflow.github.io/Tempest/api/).
```
Comment (c0ffeeca7) [inline source/_integrations/weatherflow_cloud.markdown]:
```suggestion
The **WeatherFlow Cloud** integration provides access to cloud provided Weather Forecast of a user's Tempest Weather Stations. In order to access the station, you will need to configure the integration with an [Api Key](https://weatherflow.github.io/Tempest/api/).
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @jeeftor :+1:

**Comment by davet2001** [comment]
@jeeftor After merge observation:

According to icons.json, `wind` is not a valid option. Should this be `windy`?
```
| windy | windy |
```

**Pull request merged**